packages/server/graphql/public/mutations/toggleTeamPrivacy.ts

```diff
@@ -0,0 +1,49 @@
+import toTeamMemberId from '../../../../client/utils/relay/toTeamMemberId'
+import getKysely from '../../../postgres/getKysely'
+import {getUserId, isUserOrgAdmin} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const toggleTeamPrivacy: MutationResolvers['toggleTeamPrivacy'] = async (
+  _source,
+  {teamId},
+  {authToken, dataLoader}
+) => {
+  const viewerId = getUserId(authToken)
+  const pg = getKysely()
+
+  const teamMemberId = toTeamMemberId(teamId, viewerId)
+  const teamMember = await dataLoader.get('teamMembers').load(teamMemberId)
+  if (!teamMember) {
```

Comment:
-1 Later we allow the org admin to change the setting in addition to the team lead. I think an org admin should be able to modify team settings without needing to join first.
I think an org admin also can see all private teams, can't they?

Reply:
I've created an issue for this: https://github.com/ParabolInc/parabol/issues/11015

Reply:
I think this view has a lot of clutter already and needs some tweaks. I think we could continue with the labels and add a "Private Team" bubble next to the "Member" one, in a different colour.
I also would right align the "Member Count" and " Last Met At" columns as it looks a bit messy.

I think we could introduce a ... menu to the team columns, or what I would prefer is a dedicated settings page. In fact it could be the same as in the Team Dashboard view, just nested in `/organization/.../team/...` so the org admin could access it.
I agree this should be a separate issue/PR and does not need to be done for this milestone.

Reply:
Sounds good. I've updated the issue https://github.com/ParabolInc/parabol/issues/11015 to include your comment here and to focus more on updating the org teams settings page.